rasa/core/interpreter.py

```diff
@@ -273,10 +278,17 @@ async def parse(
 
         if self.lazy_init and self.interpreter is None:
             self._load_interpreter()
+
         result = self.interpreter.parse(text)
 
         return result
 
+    def synchronous_parse_message(self, message: Message) -> Optional[Message]:
```

Comment:
`parse` should call this one

Reply:
what do you mean? 
will it be clearer what's going on if I rename it into `featurize_message`?

Reply:
ah, so the difference now. `featurize_message` would actually be a great name!

Reply:
renamed!